.github/workflows/brev-nightly-e2e.yaml

```
@@ -0,0 +1,58 @@
+  brev-nightly-e2e:
+    if: github.repository == 'NVIDIA/NemoClaw'
+    strategy:
+      fail-fast: false
+      matrix:
+        test_suite: [all, messaging-providers, full]
+    uses: ./.github/workflows/e2e-branch-validation.yaml
+    with:
+      branch: ${{ github.event_name == 'schedule' && 'main' || inputs.branch || github.ref_name }}
+      test_suite: ${{ matrix.test_suite }}
+      use_launchable: true
+      use_published_launchable: true
```

actionlint (1.7.12)

[error] input "use_published_launchable" is not defined in "./.github/workflows/e2e-branch-validation.yaml" reusable workflow. defined inputs are "branch", "brev_create_timeout_seconds", "brev_gpu_min_vram", "brev_gpu_name", "brev_gpu_type", "brev_provider", "keep_alive", "pr_number", "setup_script_url", "test_suite", "use_launchable"

(workflow-call)